### PR TITLE
common: do not log keyring secret

### PR DESCRIPTION
**Pull request by guits**
let's not display any keyring secret by default in ansible log.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1980744

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-lvm_osds

File: roles/ceph-nfs/tasks/pre_requisite_non_container.yml [hunk 1]
```diff
@@ -93,3 +94,4 @@
             owner: ceph
             group: ceph
             mode: "{{ ceph_keyring_permissions }}"
+            no_log: "{{ no_log_on_ceph_key_tasks }}"
```
Comment (dsavineau) [inline roles/ceph-nfs/tasks/pre_requisite_non_container.yml:97]:
```suggestion
          no_log: "{{ no_log_on_ceph_key_tasks }}"
```

**Review by dsavineau — CHANGES_REQUESTED** [review]

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-collocation

**Review by dsavineau — APPROVED** [review]

**Comment by dsavineau** [comment]
jenkins test centos-non_container-collocation

**Comment by dsavineau** [comment]
jenkins test centos-non_container-all_daemons

**Comment by dsavineau** [comment]
jenkins test centos-non_container-collocation

**Pull request merged**